Loenn/lang/en_gb.lang

```diff
@@ -0,0 +1,133 @@
+entities.FactoryHelper/FactoryActivatorDashBlock.placements.name.factory_activator_dash_block=Dash Block - Factory Activator
+entities.FactoryHelper/FactoryActivatorDashBlock.attributes.description.canDash=Whether the player can break the block by dashing into it.
+entities.FactoryHelper/FactoryActivatorDashBlock.attributes.description.blendin=Blends the block with the walls it touches, making it less apparent.
+entities.FactoryHelper/FactoryActivatorDashBlock.attributes.description.tiletype=Determines the visual appearance of the wall.
+entities.FactoryHelper/FactoryActivatorDashBlock.attributes.description.permanent=Once broken, the block will remain that way, even if the player dies. Also makes the factory activation permanent.
+entities.FactoryHelper/FactoryActivatorDashBlock.attributes.description.activationIds=A comma separated set of string Ids for all the Activation Ids this dash blcok should activate when broken.
+
+# Fan
+entities.FactoryHelper/FanHorizontal.placements.name.active=Fan (Horizontal, Active)
+entities.FactoryHelper/FanHorizontal.placements.name.inactive=Fan (Horizontal, Inactive)
+entities.FactoryHelper/FanHorizontal.attributes.description.activationId=String value of the entity's Activation Id. When a factory activator with this Id is turned on the entity will toggle state.
+entities.FactoryHelper/FanHorizontal.attributes.description.startActive=If `true` the entity will initially be active.
+entities.FactoryHelper/FanVertical.placements.name.active=Fan (Vertical, Active)
+entities.FactoryHelper/FanVertical.placements.name.inactive=Fan (Vertical, Inactive)
+entities.FactoryHelper/FanVertical.attributes.description.activationId=String value of the entity's Activation Id. When a factory activator with this Id is turned on the entity will toggle state.
+entities.FactoryHelper/FanVertical.attributes.description.startActive=If `true` the entity will initially be active.
+
+# Jump Thru Rusty
+entities.FactoryHelper/RustyJumpthruPlatform.placements.name.jump_thru_rusty=Jump Through (Rusty Metal)
+
+# Machine Heart
+entities.FactoryHelper/MachineHeart.placements.name.machine_heart=Machine Heart
+
+# Piston
+entities.FactoryHelper/PistonUp.placements.name.normal=Piston (Up)
+entities.FactoryHelper/PistonUp.placements.name.heated=Piston (Up, Heated)
+entities.FactoryHelper/PistonUp.attributes.description.moveTime=The time it takes for the piston head to travel between its start and end position in seconds.
+entities.FactoryHelper/PistonUp.attributes.description.pauseTime=The time the piston head waits in place after movement is done before moving again in seconds.
+entities.FactoryHelper/PistonUp.attributes.description.initialDelay=The time it takes before the piston first starts moving in seconds.
+entities.FactoryHelper/PistonUp.attributes.description.startActive=If `true` the entity will initially be active.
+entities.FactoryHelper/PistonUp.attributes.description.heated=Whether the pipe is heated or not.
```

Comment:
Would be nice to make clear that "heated" = grabbing it will kill you

Reply:
i'm just porting the tooltips, but since we "own" the mod we might as well improve stuff like this. + i'll back tooltips backwards too